sdf_timing/tests/data/golden/spec-example4.sdf

```diff
@@ -6,8 +6,8 @@
         (CELLTYPE "XOR")
         (INSTANCE )
         (TIMINGENV
-                (PATHCONSTRAINT I2.H01 I1.N01 (989.0:1269.0:1269.0) (989.0:1269.0:1269.0))
-                (PATHCONSTRAINT I2.H01 I3.N01 (904.0:1087.0:1087.0) (904.0:1087.0:1087.0))
+                (PATHCONSTRAINT I1.N01 I2.H01 (989.0:1269.0:1269.0) (989.0:1269.0:1269.0))
+                (PATHCONSTRAINT I3.N01 I2.H01 (904.0:1087.0:1087.0) (904.0:1087.0:1087.0))
```

Comment:
is this OK?

Reply:
Hmm, that is not right. This is a problem in `sdfwrite.emit()`. If I recall right, the original parser did incorrectly swap source and destination ports in few cases; guess that this is one of them. I fixed that in the parser but not in `emit()` (either because I did not notice or because it wasn't easily possible). After all, I think that `emit()` should be discontinued in favor of `print_sdf()`. To see difference between the two, compare outputs of `utils/sdfwriter.py` and `utils/reformat.py` on the input file in question.